packages/core/src/models/uischema.ts

```diff
@@ -244,6 +253,10 @@ export interface Categorization extends UISchemaElement {
   elements: (Category | Categorization)[];
 }
 
+export const isInternationalizable = (element: unknown): element is Required<Internationalizable> => {
```

Comment:
I would change the name a bit to better fit the interface name, i.e. `..able` -> optional prop and `...ed` -> required prop.

```suggestion
export const isInternationalized = (element: unknown): element is Required<Internationalizable> => {
```